app/views/underpayments/UnderpaymentDetailSummaryView.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(layout: templates.Layout,
+        govukSummaryList: GovukSummaryList,
+        h1: components.h1,
+        formHelper: FormWithCSRF,
+        button: components.button
+)
+@(underpaymentType: String, underpaymentSummary: SummaryList)(implicit request: Request[_], messages: Messages)
+
+@getMessage(isTitle: Boolean = true) = @{
+    val key = if(isTitle) "pageTitle" else "pageHeader"
+    val emptySpace = " "
+    val middleMessage = if(underpaymentType == "B00" | underpaymentType == "E00") {
+        messages(s"underpaymentDetails.$underpaymentType.$key.lowercase")
+    } else {
+        messages(s"underpaymentDetails.$underpaymentType.$key")
+    }
+    messages(s"underpaymentDetailsSummary.$key") + emptySpace + middleMessage + emptySpace + messages("underpaymentDetails.dutyMessageEnding")
+}
+
+@layout(
+    pageTitle = getMessage(), customBackLinkUrl = Some(controllers.underpayments.routes.UnderpaymentDetailsController.onLoad(underpaymentType))) {
```

Comment:
can you pass in the backLink please as this is likely to be dynamic in future pages.

Reply:
more efficient for what we need currently to have it like this